Makefile.js

```diff
@@ -159,7 +159,8 @@ function execSilent(cmd) {
  * @private
  */
 function generateBlogPost(releaseInfo) {
-    const output = ejs.render(cat("./templates/blogpost.md.ejs"), releaseInfo),
+    const ruleList = ls("lib/rules").map(ruleFileName => ruleFileName.replace(/\.js$/, "")).sort((ruleA, ruleB) => ruleB.length - ruleA.length);
```

Comment:
Could this be multiple lines?

And I assume the reason for sorting by length descending is to ensure the longest rule is matched first, to avoid problems when a rule name is a substring of another rule?